download.py

```diff
@@ -0,0 +1,87 @@
+import json
+import sys
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
This isn't in the standard python library, so it might be worth adding a requirements file.

Reply:
Whoops! Thanks! I was mostly using a Mac to test and I see that `dateutil` is included by Apple in a directory called `Extras`. Here's the full path: `/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/dateutil`

Reply:
I removed the dependency on `dateutil` in 80f0e0f